bin/cluster

```diff
@@ -116,6 +116,10 @@ class Cluster(object):
         playbook = "playbooks/{}/openshift-cluster/update.yml".format(args.provider)
         inventory = self.setup_provider(args.provider)
 
+        env['num_masters'] = args.masters
+        env['num_nodes'] = args.nodes
+        env['num_infra'] = args.infra
+        env['num_etcd'] = args.etcd
```

Comment:
Why does "update" need to know the number of masters, nodes, infra, etc?

Doesn't it just run against all masters, nodes, infra, etc that are currently in the cluster?

These values are needed for launch because we're actively launching new instances.